pybossa/api/__init__.py

```diff
@@ -178,6 +177,17 @@ def add_task_signature(tasks):
         for task in tasks:
             sign_task(task)
 
+@jsonpify
+@login_required
+@blueprint.route('/sendmail')
+def custom_email():
```

Comment:
If this is a testing route for email functionality, it should ensure security from public access - perhaps admin only access to the route. Additionally, a parameter could be used to provide the email address in place of a hard-coded value.

Reply:
updated. thanks